bin/local-env/includes.sh

```diff
@@ -132,3 +139,30 @@ action_format() {
 command_exists() {
 	type -t "$1" >/dev/null 2>&1
 }
+
+##
+# Docker Compose helper
+#
+# Calls docker-compose with common options.
+##
+dc() {
+	docker-compose $DOCKER_COMPOSE_FILE_OPTIONS "$@"
+}
+
+##
+# WP CLI
+#
+# Executes a WP CLI request in the CLI container.
+##
+wp() {
```

Comment:
Can this perhaps be renamed to `cli`? Otherwise it might easily get confused with (or even override) a globally installed WP-CLI which is also usually available as `wp`.

Reply:
We could; I personally think it's a bit easier to read as `wp` (as one would normally write it). I agree that it could be mistaken for a globally installed binary, but I don't think it's particularly important to differentiate between the two in this case.

This function will take precedence over a globally installed `wp` binary, but only within this script so I don't think there's any risk there inside or outside the script.

If we want to rename it, I think that's fine. What do you think?

Reply:
I guess it's fine. It would only be a problem when you'd want to use the actual global `wp` binary, which I don't think is gonna be the case in the setup script anytime soon.